src/hooks/api/getters/useApplication/useApplication.ts

```diff
@@ -0,0 +1,50 @@
+import useSWR, { mutate, SWRConfiguration } from 'swr';
+import { useState, useEffect } from 'react';
+import { formatApiPath } from '../../../../utils/format-path';
+import handleErrorResponses from '../httpErrorResponseHandler';
+
+const useApplication = (name: string, options: SWRConfiguration = {}) => {
```

Comment:
It would be nice to have interface types for the output of these hooks. Similar to: 

```

interface IUseLocationSettingsOutput {
    locationSettings: ILocationSettings;
    setLocationSettings: React.Dispatch<
        React.SetStateAction<ILocationSettings>
    >;
}

export const useLocationSettings = (): IUseLocationSettingsOutput => {
    const [locationSettings, setLocationSettings] = useGlobalState();

    return { locationSettings, setLocationSettings };
};
```